pkgs/development/libraries/nanomsg/pkgconfig.in.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/pkgconfig.in b/src/pkgconfig.in
+index d4c37522..35af803a 100644
+--- a/src/pkgconfig.in
++++ b/src/pkgconfig.in
+@@ -1,7 +1,7 @@
+ prefix=@CMAKE_INSTALL_PREFIX@
+ exec_prefix=${prefix}
+ includedir=${prefix}/include
+-libdir=${prefix}/@CMAKE_INSTALL_LIBDIR@
++libdir=@CMAKE_INSTALL_LIBDIR@
```

Comment:
If you make that `@CMAKE_INSTALL_FULL_LIBDIR@` you can submit it upstream.

Reply:
Thanks, I'll file it separately then remove this patch once upstream merges. Keeps it in sync with #181821

Reply:
Filed nanomsg/nanomsg#1085, now we wait.